File: 백준/Bronze/10430. 나머지/나머지.java
import java.util.Scanner;
public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        int a = in.nextInt(), b = in.nextInt(), c = in.nextInt();
        System.out.printf("%d\n%d\n%d\n%d",(a+b)%c ,((a%c)+(b%c))%c , (a*b)%c, (a%c)*(b%c)%c);
    }
}

File: 백준/Bronze/10430. 나머지/나머지.kt
import java.util.Scanner
fun main(){
    val sc = Scanner(System.`in`)
    var a = sc.nextInt(); var b = sc.nextInt(); var c = sc.nextInt()
    print("${(a+b)%c}\n${((a%c)+(b%c))%c}\n${(a*b)%c}\n${((a%c)*(b%c)%c)}")
}